src/Http/Controllers/SearchController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Rapidez\Core\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Str;
+
+class SearchController
+{
+    public function __invoke(Request $request)
+    {
+        if ($searchQuery = Cache::rememberForever('search.query.' . Str::slug($request->q), function () use ($request) {
+            $searchQueryModel = config('rapidez.models.search_query');
+
+            return $searchQueryModel::where('query_text', $request->q)
```

Comment:
Does magento do an exact match by default or some kind of `LIKE` query?

Reply:
Exact. When there is no `search_query` found it creates it in default Magento. We should have this covered with my latest change.